lib/glific/communications/message.ex

```diff
@@ -443,12 +443,11 @@ defmodule Glific.Communications.Message do
 
   @spec get_group_id(map()) :: non_neg_integer() | nil
   defp get_group_id(%{provider: "maytapi"} = message_params) do
-    with %Group{id: id} <-
-           Repo.get_by(Group, %{bsp_id: message_params.group_id},
-             organization_id: message_params.organization_id
-           ) do
-      id
-    end
+    WhatsappGroup.create_or_get_group_id(
```

Comment:
pass map instead of individual parameter, code is easier to scale that way when new parameters come in

Reply:
Removed this function and instead add `maybe_create_group` in groups.ex